### PR TITLE
Fix perf1

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Fix libunwind test
1-as old url dead so changing url to github which is recent and working
2-  add default configure option as without YAML also test works
3- fix python3 issue

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by narasimhan-v** [comment]
@PraveenPenguin please add additional reviewers, if needed

**Comment by PraveenPenguin** [comment]
@narasimhan-v I don't think so addition reviewer needed :)

File: perf/libunwind.py [hunk 1]
```diff
@@ -48,18 +48,29 @@ def setUp(self):
...
+        https://github.com/libunwind/libunwind
         '''
+        configure_option = ''
+        if cpu.get_vendor() == 'intel':
+            configure_option = 'CC=icc CFLAGS="-g -O3 -ip" CXX=icc CCAS=gcc ' \
+                               'CCASFLAGS=-g LDFLAGS="-L$PWD/src/.libs"'
+        elif cpu.get_vendor() == 'ibm':
+            configure_option = 'CFLAGS="-g -O2 -m64" CXXFLAGS="-g -O2 -m64"'
+        else:
+            self.cancel(
```
Comment (narasimhan-v) [inline perf/libunwind.py:68]: Why are we cancelling before reading yaml parameter ?

Comment (PraveenPenguin) [reply perf/libunwind.py:68]: ah right let me see how I can tweak this

**Comment by PraveenPenguin** [comment]
@narasimhan-v added that check, please have a look

File: perf/libunwind.py [hunk 2]
```diff
@@ -48,18 +48,32 @@ def setUp(self):
...
         '''
         For configure options on different architecture please refer
-        https://github.com/pathscale/libunwind
+        https://github.com/libunwind/libunwind
         '''
         configure_option = self.params.get('configure_option',
-                                           default='configure_option')
+                                           default=None)
+
+        if configure_option:
```
Comment (narasimhan-v) [inline perf/libunwind.py:64]: Is this check necessary ?

**Pull request merged**